app/views/foreman_leapp/job_templates/remediation.erb

```diff
@@ -12,6 +12,11 @@ template_inputs:
   input_type: user
   value_type: plain
   description: List of remediation ids
+- name: run_preupgrade
+  description: Run preupgrade check again when remediation entries are fixed.
+  input_type: user
+  required: true
+  options: "true\nfalse"
 %>
 
-<%= build_remediation_plan(input('remediation_ids').split(','), @host) %>
+<%= build_remediation_plan(input('remediation_ids').split(','), @host, input('run_preupgrade')) %>
```

Comment:
How about we keep the `build_remediation_plan` as is and handle the following preupgrade in the template?